types/react-big-calendar/index.d.ts

```diff
@@ -230,8 +236,8 @@ export class DateLocalizer {
     format(value: FormatInput, format: string, culture: Culture): string;
 }
 
-export interface BigCalendarProps<TEvent extends Event = Event, TResource extends object = object> extends React.Props<BigCalendar<TEvent, TResource>> {
-    localizer: DateLocalizer;
+export interface BigCalendarProps<TEvent extends Event = Event, TResource extends object = object> {
+    localizer?: DateLocalizer;
```

Comment:
Localizer doesn't seem to be optional:
https://github.com/intljusticemission/react-big-calendar/blob/master/src/Calendar.js#L57

Reply:
You're right! 
In the same file the `localizer` is passed in `mergeWithDefaults()` [here].(https://github.com/intljusticemission/react-big-calendar/blob/master/src/Calendar.js#L785) This function does not seem to handle undefined instances (check [here](https://github.com/intljusticemission/react-big-calendar/blob/master/src/localizer.js#L46)